README.md

```diff
@@ -47,6 +51,14 @@ proxy:
 
 If you're running Self-Hosted OpsLevel, replace `target` with your URL.
 
+### Set Up the Base OpsLevel URL
+
+```yaml
+opslevel:
+  baseUrl: 'https://app.opslevel.com'
```

Comment:
Do we have this as a PR on our staging deploy already or is that coming in a follow-up when we bump the version?

Reply:
We do now! Since this is a public repo, I'll share it elsewhere.